docs/src/main/sphinx/admin/properties.rst

```diff
@@ -14,6 +14,7 @@ may be used to tune Trino or alter its behavior when required.
     Spilling <properties-spilling>
     Exchange <properties-exchange>
     Task <properties-task>
+    Write partitioning <properties-write-partitioning>
```

Comment:
Maybe "writer partitioning"? Write partitioning suggest it's about layout on disk

Reply:
I would agree that "partitioning" is confusing, for exactly that reason, but I titled the document "write partitioning" since that matches the property names `use-preferred-write-partitioning` and the description of the properties:

> Enable preferred write partitioning

Using the title "writer partitioning" would be inconsistent and seems worse.

(As for whether "preferred write partitioning" was a bad name or not is a separate discussion ... I'm guessing that was a long discussion and this was the best we could do)

Reply:
If you think we should change all the descriptions and possibly the config names, let's follow up. It's best to avoid renaming documentation file names since that breaks links later, so we should get this right in this release.